virttest/utils_hotplug.py

```diff
@@ -483,3 +485,41 @@ def vcpuhotunplug_unsupport_str():
         return "not currently supported"
     else:
         return ""
+
+
+def create_mem_xml(params):
+    """
+    Create memory device xml.
+    """
```

Comment:
docstring for the params and return can be added

Reply:
Added doc string.